modules/server/src/main/scala/higherkindness/compendium/Main.scala

```diff
@@ -33,15 +37,40 @@ object Main extends IOApp {
 
 object CompendiumStreamApp {
 
-  def stream[F[_]: ConcurrentEffect: Timer]: Stream[F, ExitCode] =
+  def stream[F[_]: ContextShift: ConcurrentEffect: Timer]: Stream[F, ExitCode] = {
     for {
       conf <- Stream.eval(
         Effect[F].delay(pureconfig.loadConfigOrThrow[CompendiumConfig]("compendium")))
+      transactor <- Stream.resource(createHikariTransactor[F](conf))
       implicit0(storage: Storage[F])                     = FileStorage.impl[F](conf.storage)
-      implicit0(dbService: DBService[F])                 = FileDBService.impl[F]
       implicit0(utils: ProtocolUtils[F])                 = ProtocolUtils.impl[F]
+      implicit0(db: DBService[F])                        = PgDBService.impl[F](transactor)
```

Comment:
Where are this `implicit0` values used? If they are only used in one place, I would rather pass them explicitly.